web/src/ui_report.ts

```diff
@@ -39,7 +39,10 @@ export function error(
     status_box: JQuery,
     remove_after?: number,
 ): void {
-    const msg = xhr ? channel.xhr_error_message(response_html, xhr) : response_html;
+    let msg = xhr ? channel.xhr_error_message(response_html, xhr) : response_html;
+    if (xhr?.status === 400 && msg.endsWith("Read receipts are disabled in this organization.")) {
```

Comment:
I don't believe this is the correct approach, as this error is translated into numerous other languages.